src/wms/src/test/java/org/geoserver/wms/LegendNonValidCapabilitiesTest.java

```diff
@@ -111,8 +110,7 @@ public void testCapabilities_1_1_1() throws Exception {
         assertXpathEvaluatesTo(LEGEND_FORMAT, legendUrlPath + "/Format", dom);
         assertXpathEvaluatesTo(
                 BASE
-                        + "/wms?request=GetLegendGraphic&format=image%2Fpng&width=20"
-                        + "&height=20&layer=gs%3Awatertemp",
+                        + "/wms?request=GetLegendGraphic&format=image%2Fpng&width=20&height=20&layer=gs%3Awatertemp",
```

Comment:
Is there an actual change here? Looks like a reformat only.

Reply:
fixed